recipes/botan/all/conanfile.py

```diff
@@ -0,0 +1,311 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanException, ConanInvalidConfiguration
+from conans.model.version import Version
+
+
+class BotanConan(ConanFile):
+    name = 'botan'
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/randombit/botan"
+    license = "BSD-2-Clause"
+    exports = ["patches/*"]
+    description = "Botan is a cryptography library written in C++11."
+    topics = ("cryptography", "crypto", "C++11", "tls")
+    settings = 'os', 'arch', 'compiler', 'build_type'
+    options = {
+        'amalgamation': [True, False],
+        'bzip2': [True, False],
+        'debug_info': [True, False],
+        'openssl': [True, False],
+        'quiet': [True, False],
+        'shared': [True, False],
+        'fPIC': [True, False],
+        'single_amalgamation': [True, False],
+        'sqlite3': [True, False],
+        'zlib': [True, False],
+        'boost': [True, False],
+        'enable_modules': "ANY",
+        'system_cert_bundle': "ANY"
+    }
+    default_options = {'amalgamation': True,
+                       'bzip2': False,
+                       'debug_info': False,
+                       'openssl': False,
+                       'quiet': True,
+                       'shared': True,
+                       'fPIC': True,
+                       'single_amalgamation': False,
+                       'sqlite3': False,
+                       'zlib': False,
+                       'boost': False,
```

Comment:
Better name it `with_boost`

Reply:
haha this is an old discussion that I have with @SSE4

Reply:
Well, I have no stake in this discussion. I went with the `with_` option now, since it reflects the name of the option used in Botan's own build system. Should you all decide that the variant without `with_` is better, it's easy to drop the commit ;)